network/src/services/dump_peer_store.rs

```diff
@@ -25,16 +27,24 @@ impl DumpPeerStoreService {
         }
     }
 
+    #[cfg(not(target_family = "wasm"))]
     fn dump_peer_store(&self) {
         let path = self.network_state.config.peer_store_path();
         self.network_state.with_peer_store_mut(|peer_store| {
             if let Err(err) = peer_store.dump_to_dir(&path) {
-                warn!("Dump peer store error, path: {:?} error: {}", path, err);
+                ckb_logger::warn!("Dump peer store error, path: {:?} error: {}", path, err);
```

Comment:
why use fullpath here and remove the warn from `use` in line 2?

Reply:
wasm target will warn unused import `warn!`